crates/ironrdp-server/src/display.rs

```diff
@@ -231,3 +293,43 @@ pub trait RdpServerDisplay: Send {
         debug!(?layout, "Requesting layout")
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{BitmapUpdate, Framebuffer};
+    use core::num::NonZeroU16;
+    use ironrdp_graphics::{diff::Rect, image_processing::PixelFormat};
```

Comment:
issue: Same as https://github.com/Devolutions/IronRDP/pull/733#discussion_r2022990632

Reply:
yeah.. it's a tough requirement.. this is unit-testing the internal framebuffer state after an update. Since tests are disabled this isn't even built.

Reply:
If you want, we can merge as-is, and I’ll look into that

Reply:
sure, wfm -- there are certainly many things to be improved :) thanks